CONTRIBUTING.md

```diff
@@ -42,6 +42,11 @@ Before getting started, you should install some developer tools. These are not
 required to deploy Conjur but they will let you develop using a standardized,
 expertly configured environment.
 
+
+##### Preventing Secret Leaks
+Pushing to github is a form of publication, especially when using a public repo. It is a good idea to use a hook to check for secrets before pushing code.
+Follow this [link](https://github.com/cyberark/community/blob/master/Conjur/conventions/git-tips-and-tricks.md#preventing-leaks) on how to go about configuring git check for secrets before every push.
```

Comment:
this is in a weird spot, it's inserted between the prerequisite intro sentence and the list of prerequisites, and it has 5 hashtags when the parent section has 2 (so this subsection should just have 3)

can you please:
- move it down below the prerequisites
- change the number of `#` in the title
- add it to the TOC